hooks/pg_gw_utils.py

```diff
@@ -144,6 +122,37 @@ def determine_packages():
     return pkgs
 
 
+def docker_dependencies():
```

Comment:
I'm not sure about how easy/difficult is this. Will remove the function if that don't take long to merge